vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java

```diff
@@ -474,59 +369,13 @@ public void start() {
      * @return true if the client has some work to be done, false otherwise
      */
     private boolean isActive() {
-        return !getMessageHandler().isInitialUidlHandled() || isWorkPending()
-                || getMessageSender().hasActiveRequest()
-                || isExecutingDeferredCommands();
+        return false;
     }
 
     private native void initializeTestbenchHooks(
             ComponentLocator componentLocator, String ttAppId)
```

Comment:
Can't this method be removed? Only reference to it seems to be on the `init` method.

Reply:
Done